mmv1/third_party/terraform/services/privilegedaccessmanager/data_source_privileged_access_manager_entitlement.go

```diff
@@ -0,0 +1,44 @@
+package privilegedaccessmanager
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func DataSourceGooglePrivilegedAccessManagerEntitlement() *schema.Resource {
```

Comment:
you'll need to also add this to handwrittenDatasources in magic-modules/mmv1/third_party/terraform/provider/provider_mmv1_resources.go.erb - sorry for not catching this last time!